java-client/src/test/java/org/opensearch/client/opensearch/core/DeleteByQueryRequestTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
For new files you don't need to include it. 

Please also confirm that you did not copy this from non-APLv2 code.

Reply:
Removed it. Thanks